js/script.js

```diff
@@ -0,0 +1,68 @@
+
+const userInput = document.querySelector("#exampleDataList");
+const searchBtn = document.querySelector("#button-addon2");
+const searchBox = document.querySelector(".searchInputBox")
+const list = document.querySelector(".resultList");
+const loadingSpinner = document.querySelector(".loadingSpinner");
+const marqueeDiv = document.querySelector(".marqueeTextContainer");
+const marqueeLoading = document.querySelector(".loadingMarquee");
+let image;
+
+searchBtn.addEventListener('click', generateSearchList)
```

Comment:
the eventlistener should be added only after the decleration of generateSearchList(). dont trust hoisting